Apps/Medication/test/unit/Properties/launchSettings.json

```diff
@@ -10,7 +10,7 @@
       "environmentVariables": {
         "ASPNETCORE_ENVIRONMENT": "Development"
       },
-      "applicationUrl": "https://localhost:5001;http://localhost:5000"
+      "applicationUrl": "https://0.0.0.0:5001;http://0.0.0.0:5000"
```

Comment:
Since we're fixing this we should remove the 5001 port as we don't support HTTPS locally.

Reply:
agreed, its removed!